index.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+?>
+
+<h1>Portfolio Edit Page</h1>
+    <br><br>
+    <h2>Header Nav Bar Links</h2>
+    <br><br>
+<form method="post" action="admin.php">
+    Type: <select name="type">
+        <option value="home">Home</option>
+        <option value="about me">About Me</option>
+        <option value="portfolio">Portfolio</option>
+        <option value="home">Contact</option>
+    </select>
+    Link url: <input type="url" name="link url">
+    </form>
+    <input type="submit">
+</form>
+
+<h2>About Me</h2>
```

Comment:
Maybe consider div's (or equivalent) to separate out the page

Reply:
added